### DISCUSSION TITLE
Saves in separate folder

### DISCUSSION THREAD

**Discussion by calhix** [Keriew/augustus#460] [Ideas]
It would be nice if we could have a saves folder within the install directory/be able to choose a saves folder location. This is especially helpful for those that like to sync their saves files (such as with syncthing) across devices so we can just continue our game.

**Comment by crudelios** [comment]
Hey, this is a planned feature,  but I haven't gotten around to implement it yet.

**Comment by calhix** [comment]
Cool, thanks for all the hard work! Super enjoyable.

Sent from ProtonMail mobile

-------- Original Message --------
